docs/getting-started/es/README.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
If i'm reading the VuePress docs correctly, I think the file structure will need to be adjusted to `docs/es/getting-started/README.md`.